docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -735,6 +735,22 @@ The output of the query has the following columns:
     - ``array(integer)``
     - The set of field IDs used for equality comparison in equality delete files
 
+``$all_files`` table
+^^^^^^^^^^^^^^^^^^^^
+
+The ``$all_files`` table provides a detailed overview of the data file in all snapshots of the Iceberg table.
+The output of this system table can contain duplicate entries, as corresponding snapshots can refer to the same data files.
+This table is handy to debug the data issues, where the snapshot is rolled back/ bring back to the future from the Trino/Spark
```

Comment:
bring back -> brought back

Reply:
Just remove the "/ brought back"

Reply:
also probably remove ... to the future from the Trino/Spark

Reply:
updated @mosabua